socorro/external/es/crashstorage.py

```diff
@@ -520,7 +520,7 @@ class ESCrashStorageRedactedJsonDump(ESCrashStorageRedactedSave):
         default="socorro.external.crashstorage_base.Redactor",
         from_string_converter=class_converter,
     )
-    required_config.add_option(
+    required_config.es_redactor.add_option(
```

Comment:
I'm 90% positive these had the wrong namespace and thus weren't doing anything. The redactor works currently since the configuration is done in the processor app in a giant configuration thing.